drivers/clock_control/clock_control_wch_rcc.c

```diff
@@ -34,6 +34,12 @@
 #define WCH_RCC_SRC_IS_HSI 1
 #endif
 
+#if DT_NODE_HAS_COMPAT_STATUS(DT_NODELABEL(clk_lse), wch_ch32v00x_hse_clock, okay)
+#define WCH_HSE_BYPASS DT_PROP(DT_NODELABEL(clk_lse), lse_bypass)
```

Comment:
The property is defined on clk_hse, and has the name "hse-bypass", which is different to the "lse_*" here. Is that intended?